### DISCUSSION TITLE
下载 geoip.dat和geosite.dat失败

### DISCUSSION THREAD

**Discussion by 00BUG00** [v2rayA/v2rayA#1410] [Q&A]
在ubuntu中使用snap商店安装 v2rayA后  打开页面一直 提示下载 geoip.dat和geosite.dat ，但是一直下载失败，从其他地方下载这两个文件后，应该把这两个文件放到哪个路径下？

**Comment by jiange1236** [comment]
/usr/share/v2ray 看看

**Reply by RX78-3** [reply]
我是Debian上面的docker版的，连这个文件夹都没有，自己建了也不读，还有/usr/share/xray也是

**Reply by jiange1236** [reply]
docker 版在docker容器里面，要 docker exec -it v2raya /bin/bash 进去看

**Reply by RX78-3** [reply]
可是我开root权限也进不去啊
OCI runtime exec failed: exec failed: unable to start container process: exec: "/bin/bash": stat /bin/bash: no such file or directory: unknown

**Reply by jiange1236** [reply]
> 可是我开root权限也进不去啊 OCI runtime exec failed: exec failed: unable to start container process: exec: "/bin/bash": stat /bin/bash: no such file or directory: unknown

试试docker exec -it v2raya /bin/sh

**Comment by wxyShine** [comment]
![image](https://github.com/v2rayA/v2rayA/assets/42802152/74cf4901-8be3-4559-ab08-2f5b8ce25489)
![image](https://github.com/v2rayA/v2rayA/assets/42802152/f23bfc4b-5107-4eff-8611-4d39faed8e91)

**Reply by ZHAN2006** [reply]
```
wget https://ghproxy.net/https://raw.githubusercontent.com/v2rayA/dist-v2ray-rules-dat/master/gfw.txt -O /usr/share/xray/gfw.txt
ln -s /usr/share/xray/gfw.txt /run/user/0/v2raya/gfw.txt
```

**Comment by qux-bbb** [comment]
试过手动放进去不好用，可以用这里的方法一通过软件源安装  
https://v2raya.org/docs/prologue/installation/debian/#%E6%96%B9%E6%B3%95%E4%B8%80%E9%80%9A%E8%BF%87%E8%BD%AF%E4%BB%B6%E6%BA%90%E5%AE%89%E8%A3%85

**Comment by gsssz** [comment]
2025/01/17 04:55:58.616  [E] [version.go:39]  failed to get latest version of GFWList: Get "https://hubmirror.v2raya.org/api/v2rayA/dist-v2ray-rules-dat/tags": dial tcp: lookup hubmirror.v2raya.org on 127.0.0.1:53: no such host

域名挂了?
